src/bp-templates/bp-nouveau/js/buddypress-activity-post-form.js

```diff
@@ -2114,7 +2114,7 @@ window.bp = window.bp || {};
 											cleanTags: [ 'meta', 'div', 'main', 'section', 'article', 'aside', 'button', 'svg', 'canvas', 'figure', 'input', 'textarea', 'select', 'label', 'form', 'table', 'thead', 'tfooter', 'colgroup', 'col', 'tr', 'td', 'th', 'dl', 'dd', 'center', 'caption', 'nav' ],
 											unwrapTags: [ 'ul', 'ol', 'li' ]
 										},
-										imageDragging: false
+										imageDragging: true
```

Comment:
@kishanjasani we not supporting image dropping in editor as we have media uploading.

Reply:
Did changes, As per your Suggestion.